pihole-google.txt

```diff
@@ -107,11 +108,13 @@ www.googletagmanager.com
 www.googletagservices.com
 wwwctp.googletagmanager.com
 youtube.googleapis.com
+www.google.com/amp
```

Comment:
```suggestion
```

Please remove the part after the /, the list only allows (sub)domains.

Reply:
Done